Code/GraphMol/MolDraw2D/MolDraw2DDetails.h

```diff
@@ -110,21 +110,27 @@ inline std::string formatDouble(double val) {
   return boost::str(boost::format("%.1f") % val);
 }
 
-bool doesLineIntersect(const StringRect &rect, const Point2D &end1,
-                       const Point2D &end2, double padding);
+RDKIT_MOLDRAW2D_EXPORT bool doesLineIntersect(const StringRect &rect,
+                                              const Point2D &end1,
+                                              const Point2D &end2,
+                                              double padding);
 // returns true if any corner of triangle is inside the rectangle.
-bool doesTriangleIntersect(const StringRect &rect, const Point2D &pt1,
-                           const Point2D &pt2, const Point2D &pt3,
-                           double padding);
-bool doesLineIntersectEllipse(const Point2D &centre, double xradius,
-                              double yradius, double padding,
-                              const Point2D &end1, const Point2D &end2);
+RDKIT_MOLDRAW2D_EXPORT bool doesTriangleIntersect(const StringRect &rect,
+                                                  const Point2D &pt1,
+                                                  const Point2D &pt2,
+                                                  const Point2D &pt3,
+                                                  double padding);
+RDKIT_MOLDRAW2D_EXPORT bool doesLineIntersectEllipse(
+    const Point2D &centre, double xradius, double yradius, double padding,
+    const Point2D &end1, const Point2D &end2);
 // angles expected in degrees, between 0 and 360.
-bool doesLineIntersectArc(const Point2D &centre, double xradius, double yradius,
-                          double start_ang, double stop_ang, double padding,
-                          const Point2D &end1, const Point2D &end2);
-bool doLinesIntersect(const Point2D &l1s, const Point2D &l1f,
```

Comment:
doLinesIntersect is the one that has been breaking the CI builds.  I thought I had added the export flag but perhaps it hasn't been merged into master yet for a different reason.  I certainly had to do it again yesterday.